{{cookiecutter.app_name}}/README.md

```diff
@@ -45,10 +45,10 @@ Run the following commands to bootstrap your environment if you are unable to ru
 ```bash
```

Comment:
Maybe consider adding an optional block above the code example reminding pipenv users that it must be installed first. Possibly:

```
{%- if cookiecutter.use_pipenv == "yes" %}
Make sure to first [install pipenv](https://pipenv.readthedocs.io/en/stable/install/#installing-pipenv).
{%- endif %}
```